src/store.js

```diff
@@ -11,6 +11,7 @@ export const useAppStore = defineStore('app', {
       recent_locations: [],
       language: localStorage.getItem('user-locale') || import.meta.env.VITE_DEFAULT_LOCALE,  // 'en'
       country: import.meta.env.VITE_DEFAULT_COUNTRY,  // 'FR',
+      currencies: [import.meta.env.VITE_DEFAULT_CURRENCY],  // ['EUR'],
```

Comment:
```suggestion
      favorite_currencies: [import.meta.env.VITE_DEFAULT_CURRENCY],  // ['EUR'],
```
of "often used" or something like this ? both in the variable and in the UI